### ISSUE TITLE
Erigon seems stuck after committed state message

### ISSUE THREAD

**Issue by mauricio-seii** [ledgerwatch/erigon#5143]
Hi,

Currently, I'm trying to run an Erigon node v2022.08.22, connecting to the Ropsten network, and synchronizing from scratch. 
The node starts up fine but after some time processing, it seems to get stuck right after printing this logging line:

```
2022-08-22T17:54:05.063215506Z [INFO] [08-22|17:54:05.063] Committed State                          gas reached=549757269943 gasTarget=549755813888
```
So here's my question, should I leave it be, or do I need to take any action to be able to continue to process blocks?

### System information
version: `v2022.08.02`
network: `ropsten`
platform: `Kubernetes`

### Expected behavior
Continue to sync to the network

**Comment by AskAlexSharov** [comment]
More logs plz

**Comment by AskAlexSharov** [comment]
add --pprof show output of http://localhost:6060/debug/pprof/goroutine?debug=1

**Comment by AskAlexSharov** [comment]
Ropsten - network already passed TheMerge event - do you use ConsensusLayer service?

**Comment by mauricio-seii** [comment]
> Ropsten - network already passed TheMerge event - do you use ConsensusLayer service?

No, we wanted this node to be synced with the network before connecting it to a CL client.

**Comment by AskAlexSharov** [comment]
I think this is not possible. @yperbasis , right?

**Comment by mauricio-seii** [comment]
hang on, let me try again. We expected the node to be syncing up to around the 12m mark, and once we got notice of the merge event we'd update the config and set up the integration to a CL.

What you are saying is that, if I'm starting a new Erigon node I should immediately connect it to a CL is that correct?

**Comment by yperbasis** [comment]
For networks that have already passed the Merge, it's best to drive Erigon by CL from the start. The alternative that you're describing, namely to first sync all pre-Merge blocks, relies on block gossip (at least when Erigon block snapshots end before the Merge). EIP-3675 [prohibits](https://eips.ethereum.org/EIPS/eip-3675#network) block gossip after the Merge, rendering this strategy brittle and depending on a small fraction of peers who are left not updated with Ropsten Terminal Total Difficulty and still gossiping. Moreover, with PR #5172 such sync without a CL node will be disabled.

TL;DR: Connect your Erigon node to a CL from the start.

**Comment by mauricio-seii** [comment]
awesome thank you very much @yperbasis and @AskAlexSharov!
